Gemfile

```diff
@@ -51,6 +51,8 @@ group :development, :test do
   gem "debug", platforms: %i[ mri windows ]
   gem "brakeman", require: false
   gem "rubocop-rails-omakase", require: false
+  gem "simplecov", require: false
+  gem "dotenv-rails"
```

Comment:
I think `simplecov` should only be in the `:test` group right?

Also, I recently moved `dotenv-rails` to the `:development` group only to avoid unexpected, failing tests when a dev has their `.env` setup incompatible with the test expectations.  I think this will still work if we pass the ENV to the test command right?

Reply:
Yes, this should be in the test group and it works if we pass the env from the CMD. Working on changes